apps/someone-else/views/personal-details.html

```diff
@@ -1,69 +1,12 @@
-{{<layout}}
-  {{$step}}Step 3 of 5{{/step}}
-
-  {{$journeyHeader}}
-    {{#t}}journeys.someone-else.header{{/t}}
-  {{/journeyHeader}}
-
-  {{$validationSummary}}
-    {{> partials-validation-summary}}
-  {{/validationSummary}}
-
-  {{$propositionHeader}}
-    {{> partials-navigation}}
-  {{/propositionHeader}}
-
-  {{$title}}
-    {{#t}}journeys.someone-else.personal-details.header{{/t}}
-  {{/title}}
-
-  {{$header}}
-    {{<partials-page-header}}
-      {{$page-header}}
-        {{#t}}journeys.someone-else.personal-details.header{{/t}}
-      {{/page-header}}
-    {{/partials-page-header}}
-  {{/header}}
-
-  {{$content}}
-    <p>{{#t}}journeys.someone-else.personal-details.subheader{{/t}}</p>
-
-    {{<partials-form}}
-
-      {{$form}}
-        {{#input-text}}fullname{{/input-text}}
-
-        <fieldset id="date-of-birth-group" class="{{#errors.date-of-birth}}validation-error{{/errors.date-of-birth}} form-group">
-
-          <legend>
-            <span class="form-label-bold">{{#t}}fields.date-of-birth.legend{{/t}}</span>
-          </legend>
-
-          {{#errors.date-of-birth}}
-            <span id="date-of-birth-error" class="error-message">
-              {{errors.date-of-birth.message}}
-            </span>
-          {{/errors.date-of-birth}}
-
-          <div class="form-group form-date">
-            <p id="date-hint" class="form-hint">
-              {{#t}}fields.date-of-birth.hint{{/t}}
-            </p>
-            {{#input-date}}date-of-birth{{/input-date}}
-          </div>
-
-        </fieldset>
-
-        {{#select}}nationality{{/select}}
-
-        {{#input-text}}passport{{/input-text}}
-
-        {{#input-submit}}continue{{/input-submit}}
-
-      {{/form}}
-
-    {{/partials-form}}
-
-  {{/content}}
-
-{{/layout}}
+{{<partials-page}}
+{{$journeyHeader}}
+{{#t}}journey.someone-else.header{{/t}}
+{{/journeyHeader}}
+{{$page-content}}
+<p>{{#t}}pages.personal-details.subheader{{/t}}</p>
+{{#fields}}
+{{#renderField}}{{/renderField}}
+{{/fields}}
+{{#input-submit}}continue{{/input-submit}}
+{{/page-content}}
+{{/partials-page}}
```

Comment:
last time I'll mention indentation 😄  I guess you can either explain it or address it

Reply:
😄 as above Have fixed indentation as suggested (added as a separate commit just for clarity - I will squash before merging)